packages/core/src/components/chip/index.js

```diff
@@ -0,0 +1,78 @@
+class Chip {
+  static instances = new WeakMap();
+
+  static get cssClasses() {
+    return {
+      ACTIVE: 'ray-chip--active'
+    };
+  }
+
+  static get strings() {
+    return {
+      INIT_SELECTOR: '.ray-chip'
+    };
+  }
+
+  static create(element, options) {
+    return this.instances.get(element) || new this(element, options);
+  }
+
+  static createAll(target = document, _options = { active: false }) {
+    // Finds all instances of select on the document or within a given element and instantiates them.
+    const options = {
+      initSelector: this.strings.INIT_SELECTOR,
+      ..._options
+    };
+
+    const chips = Array.from(target.querySelectorAll(options.initSelector));
+    chips.forEach(select => this.create(select, options));
+  }
+
+  constructor(root, options) {
+    this._root = root;
+    this._options = options;
+    this.constructor.instances.set(this._root, this);
+
+    this.state = {
+      active: this._root.classList.contains(this.constructor.cssClasses.ACTIVE)
+    };
+
+    this._bindEventListeners();
+    this.assignClasses();
+  }
+
+  _bindEventListeners() {
+    this._root.addEventListener('mousedown', this.onClick);
```

Comment:
this came up for another component I can't quite remember, but should we add this for `touchstart` as well? Did you test on your phone?

Reply:
great catch! it looks like both the `touch` events and `click` events retain hover styling when clicking on mobile. so i added the hover style to non mobile screens only. 

here's a lil demo https://cl.ly/16bf6c68733d